src/View/Helper/PaginatorHelper.php

```diff
@@ -47,7 +50,7 @@ public function __construct(View $View, array $config = [])
      *   using `'size' => 'lg'` would add class `pagination-lg` to UL tag.
      *
      * @param array $options Options for the numbers.
-     * @return string Numbers string.
+     * @return string|false Numbers string.
      * @link http://book.cakephp.org/3.0/en/views/helpers/paginator.html#creating-page-number-links
      */
     public function numbers(array $options = [])
```

Comment:
Currently `Paginators::numbers()` by default auto adds wrapping class and also has options to generate prev and next links. I don't really like that and would prefer to have the method behavior similar to core method. So I suggest removing these extras and instead add a new method which generates (maybe called `links()`) number links along with wrapper and prev, next, first, last links.